source/tutorials/yourfirst.rst

```diff
@@ -2475,7 +2475,7 @@ At the end of this section, we'll talk about how to publish your package to PyPI
 
 But first, let's learn how to package our code properly.
 
-1. Create a `setup.py` file one level above your `mysci` package:
+1. Create a `setup.py` file one level above your `mysci` package (in the `python_tutorial` directory):
```

Comment:
I like that.  But remember to use the `:code:` syntax, otherwise it is just italics (in RST format):

```suggestion
1. Create a :code:`setup.py` file one level above your :code:`mysci` package (in the :code:`python_tutorial` directory):
```

Reply:
Thanks for the reminder!